lib/pbench/cli/server/user_management.py

```diff
@@ -63,7 +62,7 @@ def user_command_cli(context):
 def user_create(
```

Comment:
Since we'll get OIDC roles when we authenticate, I don't see any possible value in caching them locally. They might get outdated, for one thing, so we just need to update them every time we decrypt a token. How is this valuable?

Reply:
If the `User` table devolves to just the UUID-to-username translation table, then I don't think there is any value to having a management interface for it.  Rows will be created when users present their access tokens to the Pbench Server, and, roughly speaking, they will never be deleted.  (I believe that we can purge any rows which aren't associated with datasets any time we want to, but the mechanism to accomplish that is more likely going to be a server housekeeping function than a (human CLI) management function.)  And, in the event that we actually need to have a human touch this table, we should be able to use direct SQL commands to do it.

On that basis, I don't see any value to this module.

Reply:
Removed this interface in the latest commit.